### PR TITLE
HederaTestConfigBuilder as factory for TestConfigBuilder

### PR DESCRIPTION
**Pull request by hendrikebbers**
Since services depends on the last platform version now we can simply use `TestConfigBuilder`. 
The changes api of the `HederaTestConfigBuilder` should make it more easy to create configurations for tests. Next to this the `HederaTestConfigBuilder` now always scans only `com.hedera` and `com.swirlds` for config records.

### CODE REVIEW

**Review by rbair23 — DISMISSED** [review]

**Review by Neeharika-Sompalli — DISMISSED** [review]

File: hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkGetAccountDetailsHandlerTest.java [hunk 1]
```diff
@@ -19,34 +19,16 @@
...
-import com.hedera.hapi.node.base.QueryHeader;
-import com.hedera.hapi.node.base.ResponseCodeEnum;
-import com.hedera.hapi.node.base.ResponseHeader;
-import com.hedera.hapi.node.base.ResponseType;
-import com.hedera.hapi.node.base.Timestamp;
-import com.hedera.hapi.node.base.TokenFreezeStatus;
-import com.hedera.hapi.node.base.TokenID;
-import com.hedera.hapi.node.base.TokenKycStatus;
-import com.hedera.hapi.node.base.TokenRelationship;
+import com.hedera.hapi.node.base.*;
```
Comment (timo0) [inline hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkGetAccountDetailsHandlerTest.java:27]: `*` import on purpose?

File: hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkGetAccountDetailsHandlerTest.java [hunk 2]
```diff
@@ -19,34 +19,16 @@
...
 import com.hedera.hapi.node.state.token.AccountApprovalForAllAllowance;
 import com.hedera.hapi.node.state.token.AccountCryptoAllowance;
 import com.hedera.hapi.node.state.token.AccountFungibleTokenAllowance;
-import com.hedera.hapi.node.token.AccountDetails;
-import com.hedera.hapi.node.token.GetAccountDetailsQuery;
-import com.hedera.hapi.node.token.GetAccountDetailsResponse;
-import com.hedera.hapi.node.token.GrantedCryptoAllowance;
-import com.hedera.hapi.node.token.GrantedNftAllowance;
-import com.hedera.hapi.node.token.GrantedTokenAllowance;
+import com.hedera.hapi.node.token.*;
```
Comment (timo0) [inline hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkGetAccountDetailsHandlerTest.java:31]: `*` import on purpose?

File: hedera-node/hedera-config/src/testFixtures/java/com/hedera/node/config/testfixtures/HederaTestConfigBuilder.java [hunk 3]
```diff
@@ -16,158 +16,85 @@
...
-import com.hedera.node.config.converter.KnownBlockValuesConverter;
-import com.hedera.node.config.converter.LegacyContractIdActivationsConverter;
-import com.hedera.node.config.converter.MapAccessTypeConverter;
-import com.hedera.node.config.converter.PermissionedAccountsRangeConverter;
-import com.hedera.node.config.converter.ProfileConverter;
-import com.hedera.node.config.converter.RecomputeTypeConverter;
-import com.hedera.node.config.converter.ScaleFactorConverter;
-import com.hedera.node.config.converter.SemanticVersionConverter;
-import com.hedera.node.config.converter.SidecarTypeConverter;
+import com.hedera.node.config.converter.*;
```
Comment (timo0) [inline hedera-node/hedera-config/src/testFixtures/java/com/hedera/node/config/testfixtures/HederaTestConfigBuilder.java:19]: `*` import on purpose?

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java [hunk 4]
```diff
@@ -16,28 +16,13 @@
...
 
-import com.hedera.hapi.node.base.AccountID;
-import com.hedera.hapi.node.base.ContractID;
-import com.hedera.hapi.node.base.Fraction;
-import com.hedera.hapi.node.base.Key;
-import com.hedera.hapi.node.base.Timestamp;
-import com.hedera.hapi.node.base.TokenID;
-import com.hedera.hapi.node.base.TokenSupplyType;
-import com.hedera.hapi.node.base.TokenType;
+import com.hedera.hapi.node.base.*;
```
Comment (timo0) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java:23]: `*` import on purpose?

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java [hunk 5]
```diff
@@ -16,28 +16,13 @@
...
 import com.hedera.hapi.node.state.common.UniqueTokenId;
-import com.hedera.hapi.node.state.token.Account;
-import com.hedera.hapi.node.state.token.AccountApprovalForAllAllowance;
-import com.hedera.hapi.node.state.token.AccountCryptoAllowance;
-import com.hedera.hapi.node.state.token.AccountFungibleTokenAllowance;
-import com.hedera.hapi.node.state.token.Nft;
-import com.hedera.hapi.node.state.token.Token;
-import com.hedera.hapi.node.state.token.TokenRelation;
+import com.hedera.hapi.node.state.token.*;
```
Comment (timo0) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java:25]: `*` import on purpose?

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java [hunk 6]
```diff
@@ -53,14 +38,7 @@
...
 import com.hedera.node.app.service.token.ReadableTokenStore;
-import com.hedera.node.app.service.token.impl.ReadableAccountStoreImpl;
-import com.hedera.node.app.service.token.impl.ReadableNftStoreImpl;
-import com.hedera.node.app.service.token.impl.ReadableTokenRelationStoreImpl;
-import com.hedera.node.app.service.token.impl.ReadableTokenStoreImpl;
-import com.hedera.node.app.service.token.impl.WritableAccountStore;
-import com.hedera.node.app.service.token.impl.WritableNftStore;
-import com.hedera.node.app.service.token.impl.WritableTokenRelationStore;
-import com.hedera.node.app.service.token.impl.WritableTokenStore;
+import com.hedera.node.app.service.token.impl.*;
```
Comment (timo0) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java:41]: `*` import on purpose?

**Review by iwsimon — DISMISSED** [review]
LGTM. After this merged, all the other code uses HederaTestConfigBuilder will need to be updated.

**Review by iwsimon — APPROVED** [review]

**Review by timo0 — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]

File: hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java [hunk 7]
```diff
@@ -147,7 +145,8 @@ void setUp() {
     /** Null arguments are not permitted to the constructor. */
     @Test
     @DisplayName("Null constructor args throw NPE")
-    @SuppressWarnings("DataFlowIssue") // Suppress the warning about null args
+    @SuppressWarnings("DataFlowIssue")
```
Comment (netopyr) [inline hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java:148]: Putting the comment in a new line makes it less clear what it is referring to. If Spotless forces us to put the comment in a new line, it should be above the relevant code.

File: hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java [hunk 8]
```diff
@@ -177,7 +176,8 @@ void nullConstructorArgsTest() {
     /** Null arguments are not permitted to the preHandle method */
     @Test
     @DisplayName("Null pre-handle args throw NPE")
-    @SuppressWarnings("DataFlowIssue") // Suppress the warning about null args
+    @SuppressWarnings("DataFlowIssue")
```
Comment (netopyr) [inline hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java:179]: Same here.

**Review by netopyr — DISMISSED** [review]
LGTM regarding `hedera-app`

**Review by timo0 — DISMISSED** [review]

**Review by netopyr — DISMISSED** [review]
LGTM in regards to hedera-app

**Review by iwsimon — APPROVED** [review]

**Review by timo0 — DISMISSED** [review]

**Review by netopyr — APPROVED** [review]
LGTM in regards to hedera-app and hedera-services

**Pull request merged**